rslib/src/notetype/mod.rs

```diff
@@ -322,25 +332,17 @@ impl Notetype {
         }
     }
 
-    /// True if any name in names does not belong to a field of this notetype.
+    /// True if any non-empty name in names does not denote a special field or
+    /// a field of this notetype.
     fn unknown_field_name<T, I>(&self, names: T) -> bool
     where
         T: IntoIterator<Item = I>,
         I: AsRef<str>,
     {
-        let special_fields = [
-            "FrontSide",
-            "Card",
-            "CardFlag",
-            "Deck",
-            "Subdeck",
-            "Tags",
-            "Type",
-        ];
         names.into_iter().any(|name| {
             // The empty field name is allowed as it may be used by add-ons.
             !name.as_ref().is_empty()
-                && !special_fields.contains(&name.as_ref())
+                && !SPECIAL_FIELDS.contains(&name.as_ref())
```

Comment:
Maybe this would be better as a lazy_static HashSet?

Reply:
(I've found myself wondering on multiple occasions where exactly the inflection point is where a set becomes cheaper, but never bothered to benchmark it before)

Reply:
Yes, that would be interesting to know. I can't imagine it makes a difference in performance here, but I guess it's more idiomatic. 🙂